### PR TITLE
LiveQuery handle unset operation

### PR DESCRIPTION
**Pull request by dplewis**
Addresses https://github.com/parse-community/parse-server/issues/4998

### CODE REVIEW

**Comment by flovilmart** [comment]
Thé op should be sent to the client as undefined is not serialized.

**Comment by dplewis** [comment]
Thats a better solution. Instead of setting undefined like in this PR. Send undefined to the client

File: src/__tests__/LiveQueryClient-test.js [hunk 1]
```diff
@@ -264,7 +264,7 @@ describe('LiveQueryClient', () => {
     expect(isChecked).toBe(true);
   });
 
-  it('can handle WebSocket response unset pointer', async () => {
+  fit('can handle WebSocket response unset field', async () => {
```
Comment (flovilmart) [inline src/__tests__/LiveQueryClient-test.js]: oops

**Comment by dplewis** [comment]
@flovilmart Thanks! Did you see my last comment?

**Comment by flovilmart** [comment]
It should likely update yes,

**Pull request merged**